cwel/component/button/button.scss

```diff
@@ -0,0 +1,136 @@
+////
+/// @group _button.scss
+/// @since 0.0.1
+////
+
+/// 1. removes focus around text in the button tag firefox
+
+button {
```

Comment:
@NathanRodger Is there a reason these styles don't apply to `.button` elements? If someone were to extend `.button` for a link in another component, they wouldn't get the styles for `button`.